polling_stations/apps/file_uploads/tests/test_api.py

```diff
@@ -122,6 +123,7 @@ def test_valid_payload_one_file(self):
         self.assertEqual(201, resp.status_code)
         self.assertEqual(1, len(Upload.objects.all()))
         self.assertEqual(1, len(File.objects.all()))
+        self.assertEqual(1, len(mail.outbox))
```

Comment:
Can you add a test case to all the tests, i.e. test that it sends in all the valid cases, and _doesn't_ send in the invalid ones.

Reply:
Done with 46ba31e457fe378096005000c0fd5af70bf5cbed